README.md

```diff
@@ -7,101 +7,60 @@ SparkPost transport for Nodemailer
 
 ## Usage
 
-Install with npm
+### Install
 
 ```
 npm install nodemailer-sparkpost-transport
 ```
 
-Require to your script
+### Create a Nodemailer transport object
 
 ```javascript
 var nodemailer = require('nodemailer');
 var sparkPostTransport = require('nodemailer-sparkpost-transport');
-```
-
-Create a Nodemailer transport object
-
-```javascript
 var transporter = nodemailer.createTransport(sparkPostTransport(options))
 ```
 
-Where
+where:
 
-  - **options** defines connection and message data
+  - **options** defines connection _default_ message properties
     - `sparkPostApiKey` - SparkPost [API Key](https://app.sparkpost.com/account/credentials). If not provided, it will use the `SPARKPOST_API_KEY` env var.
-    - `campaign_id` - Name of the campaign
-    - `content` - Content that will be used to construct a message
-    - `metadata` - Transmission level metadata containing key/value pairs
-    - `options` - JSON object in which transmission options are defined
-    - `substitution_data` - Key/value pairs that are provided to the substitution engine
-
-  For more information, see the [SparkPost API Documentation for Transmissions](https://developers.sparkpost.com/api/#/reference/transmissions)
+    - `campaign_id` - Name of the campaign (optional)
+    - `content` - Content that will be used to construct a message (optional)
```

Comment:
should we remove the content option from the docs since we want to discourage its use?